src/main/java/hu/icellmobilsoft/App.java

```diff
@@ -1,32 +1,53 @@
 package hu.icellmobilsoft;
 
-import hu.icellmobilsoft.atr.sample.parseXml;
 import hu.icellmobilsoft.atr.sample.action.SamplePatientAction;
+import hu.icellmobilsoft.atr.sample.model.Patient;
+import hu.icellmobilsoft.atr.sample.rest.parseJson;
+import hu.icellmobilsoft.atr.sample.rest.parseXml;
 
 /**
  * Hello world!
  *
  */
 public class App {
     public static void main(String[] args) {
-        parseXml opx = new parseXml();
+        // parseXml opx = new parseXml();

```

Comment:
Feladat szerint : 'módosítsuk a Main metódust, hogy az újonnan létrehozott interfacen keresztül töltse be az adatokat', itt :
  ILoadData loadData = new LoadDataImpl();
        IRequestData requestData = new RequestDataImpl();
        loadData.loadFromXml();
        loadData.loadFromJson();
majd menj végig a requestData.getAllUsersData-n és írasd ki az elemeket